packages/lit-dev-content/site/docs/templates/expressions.md

```diff
@@ -137,14 +137,21 @@ html`<main>${bodyText}</main>`
 
 Expressions in the child position can take many kinds of values:
 
-* Primitive values likes strings, numbers, and booleans
+* Primitive rendering values
+* Non-rendering values
 * `TemplateResult` objects created with the `html` function.
 * DOM nodes
 * Arrays or iterables of any of the supported types
 
-### Primitive values
+### Primitive rendering values
```

Comment:
My initial gut is that trying to coin "Primitive rendering values" as a thing with a specific meaning is more confusing than just leaving the heading "Primitive values", and saying  "With the exception of empty string, all strings numbers and booleans are converted to strings..." in the prose.

Reply:
Removed.